.github/workflows/ci.yml

```diff
@@ -402,10 +403,10 @@ jobs:
           REVISION: ${{ github.sha }}
           VERSION: ${{ steps.build-docker.outputs.version }}
         run: ${PWD}/docker/test/verify.sh '${{ steps.build-docker.outputs.image }}'
-  # Used by bors to check all tests, including the unit test matrix.
-  # New test jobs must be added to the `needs` lists!
-  # This name is hard-referenced from bors.toml; remember to update that if this name changes
   test-summary:
+    # Used by bors to check all tests, including the unit test matrix.
+    # New test jobs must be added to the `needs` lists!
+    # This name is hard-referenced from bors.toml; remember to update that if this name changes
```

Comment:
❓ in other occasions a comment is actually preceding the config it refers to, e.g. line 476-478. Is there any agreement on where to place when?

Reply:
I'm not aware of any agreement. Here I moved comments that apply to an entire job to a common place, right under the job key so that we can use code folding to hide jobs and their comments in text editors/IDEs

Reply:
yeah totally makes sense to me! 👍